### PR TITLE
Correct "doesn't exists" typos - fixes KEYCLOAK-14986

### PR DESCRIPTION
**Pull request by Elwell**
Trivial patch to correct a bunch of typos in source comments and debug output

### CODE REVIEW

File: themes/src/main/resources/theme/keycloak/common/resources/node_modules/angular-translate/dist/angular-translate.js [hunk 1]
```diff
@@ -2014,7 +2014,7 @@ function $translate($STORAGE_KEY, $windowProvider, $translateSanitizationProvide
               deferred.reject(applyNotFoundIndicators(_translationId));
```
Comment (pedroigor) [inline themes/src/main/resources/theme/keycloak/common/resources/node_modules/angular-translate/dist/angular-translate.js:2014]: This is not really our code, not sure if makes sense to change this one if they could be reverted once the library is updated.

@ssilvert makes sense?

**Review by pedroigor — DISMISSED** [review]
LGTM. Only a single comment.

File: themes/src/main/resources/theme/keycloak/common/resources/node_modules/angular-translate/dist/angular-translate.js (hunk 1 above)
Comment (stianst) [reply themes/src/main/resources/theme/keycloak/common/resources/node_modules/angular-translate/dist/angular-translate.js:2014]: Yes, this is just a copy of sources from elsewhere so we shouldn't modify them

**Comment by pedroigor** [comment]
@Elwell Are you still willing to contribute this changeset ? If so, could you please rebase?

**Review by stianst — CHANGES_REQUESTED** [review]
Remove changes to theme/keycloak/common/resources/node_modules/angular-translate/dist/angular-translate.js as these are just copies from elsewhere so we don't want to modify the source

**Comment by Elwell** [comment]
I've reverted the changes to angular-translate.js - do I need to do anything else or is that enough?

**Review by stianst — DISMISSED** [review]

**Review by stianst — APPROVED** [review]

**Review by pedroigor — APPROVED** [review]

**Comment by pedroigor** [comment]
@Elwell Could you please rebase? It should help with the test failures.

**Comment by pedroigor** [comment]
@Elwell Could you rebase, please? This should be the last action from you before merging. It looks good now.

**Pull request merged**